process/template/template.go

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) 2018 Cisco and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package template
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/ligato/cn-infra/logging"
+	"github.com/ligato/cn-infra/process/template/model/process"
+	"github.com/pkg/errors"
+)
+
+// JSONExt - JSON extension
+const JSONExt = ".json"
+
+// DefaultMode defines default permission bits for template file
+const DefaultMode = os.FileMode(int(0777))
```

Comment:
You shouldn't need to convert [untyped constants](https://golang.org/ref/spec#Constants). It's enough to defined it as: `os.FileMode(0777)`.